docs/DeveloperGuide.md

```diff
@@ -619,34 +621,33 @@ _{Explain here how the data archiving feature will be implemented}_
 **Target user profile**:
 
 * Professor from NUS School of Computing
-* Needs to manage a significant number of TA applications
+* Needs to manage a significant number of TA applications for a particular module
```

Comment:
Our app. is missing the module part where we did not take into account the interview is for which module 😢 

What's your thoughts on the updated Line 624?

Reply:
hmmm am worried about users asking why there isnt a label for the module/classification of different lists for the TA candidates, so thought we should scope it down to indicate the app should be used for a specific module at any point, so it eliminates the need for the label as the whole list will be candidates applying for a specific module.

But I see where you are coming from! let me delete this!

Reply:
I mean you made a good point too @charmainehly now that you argued it this way... UGH.. YOU DECIDE PLEASE. can make the change whichever way @charmainehly. will wait!

Reply:
It's alright! Have removed it for now :) I think the argument definitely works both ways. I can add it to the discussion points in the issue I opened up!! But I think you make sense no worries!!